### PR TITLE
Change requests/hand tracking

### PR DESCRIPTION
**Pull request by StephenHodgson**
Continued from https://github.com/XRTK/XRTK-Core/pull/476

### CODE REVIEW

File: XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/DefaultMixedRealityControllerDataProvidersProfile.asset [hunk 1]
```diff
@@ -69,30 +70,33 @@ MonoBehaviour:
...
+      type: 2}
   - instancedType:
       reference: XRTK.WindowsMixedReality.Controllers.Hands.WindowsMixedRealityHandControllerDataProvider,
         XRTK.WindowsMixedReality
     name: Windows Mixed Reality Hand Controller Data Provider
-    priority: 1
+    priority: 10
     runtimePlatform: 16
-    configurationProfile: {fileID: 11400000, guid: 8a5d0fbb07851a2409d0177ff1f998c7, type: 2}
+    configurationProfile: {fileID: 0}
```
Comment (SimonDarksideJ) [inline XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/DefaultMixedRealityControllerDataProvidersProfile.asset]: Some loss in parent profile configuration?

File: XRTK.SDK/Packages/com.xrtk.sdk/Features/UX/Scripts/Pointers/HandPointer.cs [hunk 2]
```diff
@@ -1,65 +1,41 @@
...
-        {
-            return base.TryGetPointingRay(out pointingRay);
-        }
-
-        private void InitializeHandRay()
-        {
-            SystemType handRayType = MixedRealityToolkit.Instance.ActiveProfile.InputSystemProfile.HandTrackingProfile.HandRayType;
-            handRay = (IMixedRealityHandRay)Activator.CreateInstance(handRayType.Type);
-        }
+        //private IMixedRealityHandRay handRay;
```
Comment (SimonDarksideJ) [inline XRTK.SDK/Packages/com.xrtk.sdk/Features/UX/Scripts/Pointers/HandPointer.cs]: The handray was simply removed?

Comment (FejZa) [reply XRTK.SDK/Packages/com.xrtk.sdk/Features/UX/Scripts/Pointers/HandPointer.cs]: Not sure why Stephen removed this. This was work in progress code and will most likely come back.

Comment (StephenHodgson) [reply XRTK.SDK/Packages/com.xrtk.sdk/Features/UX/Scripts/Pointers/HandPointer.cs]: Yeah I was just trying to get it to compile. Likely will come back but again I'm not a fan of having a different ray type for hands. Imho there should be just one type. I'm fine with having an interface I suppose but this is likely cruft left over from porting from mrtk.

File: XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/DefaultMixedRealityControllerDataProvidersProfile.asset (hunk 1 above)
Comment (StephenHodgson) [reply XRTK.SDK/Packages/com.xrtk.sdk/DefaultProfiles/Input/DefaultMixedRealityControllerDataProvidersProfile.asset]: No it was the wrong profile assigned

**Pull request merged**